packages/ui/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 The following is a curated list of changes in the Enact ui module, newest changes on the top.
 
+## [2.0.0-rc.1]
```

Comment:
At this time, `## [Unreleased]` would be better as we did in moonstone `CHANGELOG.md`. When releasing new version, we'll update the CHANGELOG.md with proper version.

Reply:
Thanks for your feedback. I will change it~